### PR TITLE
Audit dependencies

### PR DESCRIPTION
**Pull request by dgaubert**

### CODE REVIEW

**Comment by dgaubert** [comment]
I will need some help from @CartoDB/frontend-team to review this PR. 

cc/ @CartoDB/rt-managers

**Comment by jesusbotella** [comment]
Let me use this branch as the source for torque and do a test run. If it passes, I assume that everything will be OK as we have some tests related to Torque.

**Comment by jesusbotella** [comment]
I have just done it and all tests pass in green, so let's say that it's OK :)

Nevertheless, whenever this PR is merged, we should update CARTO.js to point to new Torque's revision and deploy the new version to Builder.

File: NEWS [hunk 1]
```diff
@@ -1,4 +1,11 @@
 2.17.1 (2018-mm-dd)
```
Comment (jesusbotella) [inline NEWS]: We should set the proper date here :)

File: dist/torque.uncompressed.js [hunk 2]
```diff
@@ -1635,6 +1635,8 @@ function GMapsTorqueLayer(options) {
 
   this.hidden = !this.options.visible;
 
+  this.showLimitErrors = options.showLimitErrors;
```
Comment (jesusbotella) [inline dist/torque.uncompressed.js]: This line seems a bit weird as we're not using it anywhere 🤔

File: NEWS (hunk 1 above)
Comment (dgaubert) [reply NEWS]: Yep, at release time! I don't want to set a date and then delay the release for any reason.

File: dist/torque.uncompressed.js (hunk 2 above)
Comment (dgaubert) [reply dist/torque.uncompressed.js]: Oops! Bad copy/paste I guess  😱

Comment (dgaubert) [reply dist/torque.uncompressed.js]: Humm, it comes from a 14 months ago commit, blame: @rubenmoya 😂 

Not sure if we should remove it. 

Have in mind that `dist` files are being regenerated like a year ago.

Comment (dgaubert) [reply dist/torque.uncompressed.js]: I'm going to leave it as it is. If it's a bug, let's fix it in another PR

**Comment by dgaubert** [comment]
@jesusbotella, after clarifying these points, can I move forward?

File: dist/torque.uncompressed.js (hunk 2 above)
Comment (jesusbotella) [reply dist/torque.uncompressed.js]: Yes, let's leave it as is then 😁

**Comment by jesusbotella** [comment]
Yes, I would say so!

File: dist/torque.uncompressed.js (hunk 2 above)
Comment (rubenmoya) [reply dist/torque.uncompressed.js]: Hi! That comes from the errors overlay we tried to add a while back, seems like we removed it from every place but torque 😂 my bad!

Comment (jesusbotella) [reply dist/torque.uncompressed.js]: Too late 😂

**Pull request merged**